package.json

```diff
@@ -178,49 +178,50 @@
         }
     },
     "scripts": {
-        "vscode:prepublish": "del-cli dist/template && webpack --mode production && cd ./src/template && node ../../node_modules/copy/bin/cli.js ./**/* ../../dist/template",
+        "vscode:prepublish": "del-cli dist/template && webpack --mode production && cd ./src/template && node ../../node_modules/copy/copy.js ./**/* ../../dist/template",
         "vscode:package": "node ./node_modules/vsce/vsce package",
         "lint": "node node_modules/tslint/bin/tslint **/*.ts -e **/*.d.ts -e node_modules/**/*.ts -p .",
         "fixlint": "node node_modules/tslint/bin/tslint **/*.ts -e **/*.d.ts -e node_modules/**/*.ts -p . --fix",
-        "compile": "del-cli dist/template && webpack --mode development && node ./node_modules/copy/bin/cli.js ./src/template/**/* dist/template",
+        "compile": "del-cli dist/template && webpack --mode development && node ./node_modules/copy/copy.js ./src/template/**/* dist/template",
         "preinstall": "npx npm-force-resolutions",
         "pretest": "npm run test-compile",
         "test": "node ./out/tests/TestsRunHost.js",
-        "test-compile": "npm run compile && del-cli out/models/recognizers/template && tsc -p ./ && cd ./src/template && node ../../node_modules/copy/bin/cli.js ./**/* ../../out/models/recognizers/template"
+        "test-compile": "npm run compile && del-cli out/models/recognizers/template && tsc -p ./ && cd ./src/template && node ../../node_modules/copy/copy.js ./**/* ../../out/models/recognizers/template"
     },
     "extensionDependencies": [
         "ms-kubernetes-tools.vscode-kubernetes-tools",
         "mindaro-dev.file-downloader"
     ],
     "devDependencies": {
-        "@types/mocha": "^5.2.7",
-        "@types/node": "^12.12.67",
-        "@types/node-fetch": "^2.5.7",
-        "@types/vscode": "1.57.0",
-        "copy": "^0.3.2",
-        "del-cli": "^4.0.1",
-        "mocha": "^6.2.2",
+        "@types/mocha": "^10.0.1",
+        "@types/node": "^18.11.17",
+        "@types/node-fetch": "^2.6.2",
+        "@types/vscode": "1.74.0",
+        "@vscode/vsce": "^2.15.0",
+        "copy": "^0.0.1",
+        "del-cli": "^5.0.0",
+        "eslint": "^8.27.0",
+        "mocha": "^10.2.0",
         "mocha-trx-reporter": "^3.3.1",
-        "ts-loader": "^6.2.1",
-        "tslint": "^5.20.1",
```

Comment:
Thanks again, I am assuming tslint was removed as part of npm audit? is that correct?

Reply:
All good, nopes audit don't do it until we make dependent rules, so here is the info which can help in connecting dots:

`tslint` is deprecated to `aslant` more here: https://www.darraghoriordan.com/2020/03/06/upgrade-tslint-deprecated-to-eslint/ 

also for other packages like: `tslint-microsoft-contrib` they still run in 4 year old packages and rules which are very outdated and now typescript is at `4.*.*` for example last satiable release for them was 4 years back. :) 

<img width="686" alt="Screenshot 2022-12-20 at 4 59 57 PM" src="https://user-images.githubusercontent.com/6233295/208657355-60b6052e-514b-4b6e-bfd6-779be1879161.png">